js/background.js

```diff
@@ -102,6 +131,17 @@ function Background() {
 
 var background = new Background();
 
+chrome.alarms.create('updateUninstallURL', {periodInMinutes: 1});
+
+chrome.alarms.onAlarm.addListener(function(updateUninstallURL){
```

Comment:
The param to the function is the actual alarm event. So you want to do something like 
```
function(alarmEvent){
    if(alarmEvent.name === "updateUninstallURL";
   .....
```